bot.py

```diff
@@ -467,6 +573,26 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def show_help(self, message, command_content):
+        """
+        Show link to full documentation
+        """
+        reply = "```\n"
+        if len(command_content) > 0:
+            reply += self.help_messages[command_content]
```

Comment:
Check if 'command_content' is in self.help_message and send a message back saying the command was not found if not.

Reply:
Help messages have been made mandatory.